### PR TITLE
Proposed title and author listing

### PR DESCRIPTION
**Pull request by timothyb0912**
Just a draft title. Let me know what y'all think and if you have any other suggestions. I'm happy to have them---especially since I'm not terribly enthused about this one. cc @bouzaghrane @hassan-obeid

### CODE REVIEW

**Comment by bouzaghrane** [comment]
@timothyb0912 @hassan-obeid Causal Graphs in Choice Modeling: The What, Why, and How.

**Comment by timothyb0912** [comment]
> @timothyb0912 @hassan-obeid Causal Graphs in Choice Modeling: The What, Why, and How.

@bouzaghrane Oooooh, I like that!

**Comment by hassan-obeid** [comment]
I like Amine's suggestion! @timothyb0912 @bouzaghrane

File: article/main.tex [hunk 1]
```diff
@@ -42,8 +42,8 @@
 \usepackage{blindtext}
 
 % Declare article info
-\title{Some Title}
-\author{All of us}
+\title{Creating, testing, and using causal graphs in choice modelling}}
+\author{Timothy Brathwaite, Mohamed Amine Bouzaghrane, Hassan Obeid}
```
Comment (hassan-obeid) [inline article/main.tex]: Suggest the ordering as "Timothy Brathwaite, Hassan Obeid, Mohamed Amine Bouzaghrane" to reflect contributions and the process of coming up with the vision/plan for the work.

Comment (timothyb0912) [reply article/main.tex]: Let's discuss this further! I'll leave the authors blank for now and merge the PR with just the changed title.

**Pull request merged**